### PR TITLE
Add command registry and descriptions

### PR DESCRIPTION
**Pull request by rrrene**
Auto-loads commands and renders usage help for CLI

### CODE REVIEW

File: lib/duty/cli.rb [hunk 1]
```diff
@@ -5,14 +5,14 @@ def initialize(args)
...
     def exec
-      stdout explain_duty if missing_command? 
+      stdout usage if missing_command?
       stdout execute_commands(@args)
     end
 
     private
 
     def stdout(string)
-      $stdout.puts strip(string)
```
Comment (rrrene) [inline lib/duty/cli.rb]: Any reason for this besides heredocs? Cause formatting the  `usage` for `CLI` was impossible with this `strip` call present ...

**Comment by rrrene** [comment]
yeah, maybe we should have coordinated that, since I just pushed half an hour of work against your changes :sob:

**Comment by JanOwiesniak** [comment]
I though you wouldn't get out of bed before noon :smile:

**Comment by JanOwiesniak** [comment]
What do you @rrrene think about my plugin approach? Good enough to merge into master? Or do you have any improvements?

**Comment by rrrene** [comment]
You mean besides `not hijacking other people's feature-branches`? :neckbeard:

**Comment by rrrene** [comment]
Jesus, after the rebase it seems like we implemented the exact same thing regarding the Registry ... Only thing that's left is my [refactoring of the command descriptions into their own method](https://github.com/JanOwiesniak/duty/commit/3690b3bd3657cff16c037cdcd13fa8d81c6d53dd)

**Comment by JanOwiesniak** [comment]
Thats kind of funny.

**Comment by rrrene** [comment]
Re: additional commands ... I think it is okay for now. The fact that the functionaliy is tight to one additional directory might be something we will need to refactor in the future, but for the first use-cases we discussed this should be more than enough :+1:

**Pull request merged**